clustering/singleton/extension/src/main/java/org/wildfly/extension/clustering/singleton/SingletonResourceDefinition.java

```diff
@@ -76,8 +80,8 @@
         ;
         private final SimpleAttributeDefinition definition;
 
-        private Attribute(String name, ModelType type, CapabilityReferenceRecorder reference) {
-            this.definition = new SimpleAttributeDefinitionBuilder(name, type)
+        Attribute(String name, ModelType type, CapabilityReferenceRecorder reference) {
+            this.definition = new SimpleAttributeDefinitionBuilder(name, type, true)
```

Comment:
But the attribute actually is required. So we'd be reporting incorrect metadata to work around a test fixture problem?